cmd/lima-guestagent/daemon_linux.go

```diff
@@ -40,16 +28,13 @@ var daemonCommand = &cli.Command{
 }
```

Comment:
Probably this ticking is too heavy for watching iptables. Instead we should wait for `NETLINK_AUDIT` events or some eBPF stuff, but I can work on that in a separate PR after merging this.